app/javascript/plugins/email_pension/EmailPensionView.js

```diff
@@ -225,7 +225,6 @@ export const mapStateToProps = ({ emailTarget, consent }) => {
     email,
     name,
     country,
-    emailSubject,
```

Comment:
This fixes the subject being passed in as props, but the name / email fields are still not being pre-populated because we're not dispatching the initialize action: https://github.com/SumOfUs/Champaign/blob/master/app/javascript/packs/email_pension.js#L36

I'll push a fix in a sec.